extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -305,12 +305,14 @@ export class AmpA4A extends AMP.BaseElement {
     this.isRelayoutNeededFlag = false;
 
     /**
-     * Used as a signal in some of the CSI pings. Canary is shortened to 'ca'
-     * and production to 'pr'. TODO(@glevitzky, #10060) Include other release
-     * types (control) when available.
+     * Used as a signal in some of the CSI pings.
      * @private @const {string}
      */
-    this.releaseType_ = isCanary(this.win) ? 'ca' : 'pr';
+    this.releaseType_ = {
```

Comment:
Do we really need two different mappings for the same thing?

Reply:
No, I suppose we don't. Switched to just one mapping now.